client/src/components/profile/components/TimeLine.tsx

```diff
@@ -341,7 +323,7 @@ class TimelineInner extends Component<
     );
   }
 }
-
+/* eslint-disable */
```

Comment:
Can specific rules be disabled here?  Turning off eslint entirely is a bit indiscriminate.